typings/index.d.ts

```diff
@@ -1231,6 +1231,8 @@ export class Interaction<Cached extends CacheType = CacheType> extends Base {
   public user: User;
   public version: number;
   public memberPermissions: CacheTypeReducer<Cached, Readonly<Permissions>>;
+  public locale: string;
+  public guildLocale: string | null;
```

Comment:
This can be narrowed with all our guild type guard checks, right?

Reply:
Good point! I'll add that